### PR TITLE
use credentials binding and fix resource scripts

### PR DESCRIPTION
**Pull request by estute**
@benpatterson additional fixes found when running the seed.

### CODE REVIEW

File: mobileApp/resources/copy_keys.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
-#!/usr/bin/bash env
+#!/usr/bin/env bash -e
```
Comment (benpatterson) [inline mobileApp/resources/copy_keys.sh]: Oh I didn't realize you could pass in the `-e` on the hashbang line. This works?

**Comment by benpatterson** [comment]
👍

File: mobileApp/resources/copy_keys.sh (hunk 1 above)
Comment (estute) [inline mobileApp/resources/copy_keys.sh]: no, just a typo

**Comment by benpatterson** [comment]
from my perspective, merge when ready

**Pull request merged**